python/lsst/afw/table/_syntax.py

```diff
@@ -202,3 +202,70 @@ def processArray(a):
         else:
             d[name] = processArray(self.get(schemaItem.key))
     return d
+
+def BaseCatalog_asAstropy(self, cls=None, copy=False, unviewable="copy"):
+    """!
+    Return an astropy.table.Table (or subclass thereof) view into this catalog.
+
+    @param[in]   cls        Table subclass to use; None implies astropy.table.Table itself.
+                            Use astropy.table.QTable to get Quantity columns.
+
+    @param[in]  copy        Whether to copy data from the LSST catalog to the astropy table.
+                            Not copying is usually faster, but can keep memory from being
+                            freed if columns are later removed from the Astropy view.
+
+    @param[in]  unviewable  One of the following options, indicating how to handle field types
```

Comment:
Yes and no.  This argument is indeed the only one we can't get through the `__astropy_table__` call.  But I also think it's not obvious that the astropy table constructor interface is the more natural one even if it did kwarg forwarding.  We had some discussion of this on the current Astropy PR's predecessor (astropy/astropy/pull/4864, but it's a long thread); it's sort of unfortunate that Python users expect overloaded constructors, even though the language doesn't actually support overloading.